src/crsAdapter/AmadeusSPCTomaAdapter.js

```diff
@@ -213,14 +213,13 @@ class AmadeusSPCTomaAdapter {
 
                 if (url.indexOf('.sellingplatformconnect.amadeus.com') > -1) {
                     this.logger.info('auto detected Amadeus URL: ' + url);
-
                     return url;
                 }
 
                 this.logger.info('could not auto detect any Amadeus URL');
             };
 
-            let connectionUrl = cleanUrl(getConnectionUrlFromReferrer() || this.connectionOptions.connectionUrl);
+            let connectionUrl = cleanUrl(this.connectionOptions.connectionUrl || getConnectionUrlFromReferrer());
```

Comment:
what is purpose of this change?

Reply:
Previously `getConnectionUrlFromReferrer ` method returned empty string and the `this.connectionOptions.connectionUrl ` was used to connect. Currently `getConnectionUrlFromReferrer` method returns URL from iframe (booking_manager url) that is invalid and there is a connection problem. Perhaps this has been changed in the latest version of the web browser and what document.referrer returns for iframe objects.